### PR TITLE
recover diagnostic s01e03 data

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: configuration/pipeline_config.json [hunk 1]
```diff
@@ -40,6 +40,13 @@
         "3903ad8e-284e-48a0-881b-89c0b06b9334",
         "45939459-e4d5-4ad6-8650-3e7134b9a035"
       ]
+    },
+    {
+      "SourceType": "RecoveryCSVURLs",
```
Comment (as2388) [inline configuration/pipeline_config.json]: Is this right? I thought this was called RecoveryCSV now

Comment (as2388) [inline configuration/pipeline_config.json]: Is this right? I thought this was called RecoveryCSV now

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -40,6 +40,13 @@
...
+      "ContactsFileName": "imaqal_covid19_contacts",
```
Comment (as2388) [inline configuration/pipeline_config.json]: The version of pipeline_config in IMAQAL_COVID19 doesn't have a ContactsFileName option

File: src/translate_rapid_pro_keys.py [hunk 3]
```diff
@@ -108,6 +108,12 @@ def remap_radio_shows(cls, user, data, pipeline_configuration):
                 user, data, remapping.time_key, remapping.show_pipeline_key_to_remap_to,
                 remapping.range_start_inclusive, remapping.range_end_exclusive, remapping.time_to_adjust_to
             )
+            # Correct diagnostic s01e03 messages recovered from the All networks downtime issue
+            cls._remap_radio_show_by_time_range(
```
Comment (as2388) [inline src/translate_rapid_pro_keys.py]: You can set this from configuration json

File: configuration/pipeline_config.json (hunk 2 above)
Comment (as2388) [inline configuration/pipeline_config.json]: The version of pipeline_config in IMAQAL_COVID19 doesn't have a ContactsFileName option

**Review by as2388 — CHANGES_REQUESTED** [review]
I don't think the configuration makes sense when comparing to the code I recently copied from this project. Did you use an out of date config file to check how to do this or am I misunderstanding the code?

File: configuration/pipeline_config.json (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: Added this from memory - Would you be having an example of a recent project you did this - that has new configurations?

Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: updated thanks.

File: configuration/pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: removed thanks.

File: src/translate_rapid_pro_keys.py (hunk 3 above)
Comment (IsaackMwenda) [reply src/translate_rapid_pro_keys.py]: moved to pipeline configuration json file.

File: configuration/pipeline_config.json [hunk 4]
```diff
@@ -40,12 +40,26 @@
...
     }
   ],
   "PhoneNumberUuidTable": {
     "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
     "TableName": "IMAQAL"
   },
+  "TimestampRemappings": [
+    {
+      "TimeKey": "received_on",
+      "ShowPipelineKeyToRemapTo":  "diagnostic_s01e03_raw",
```
Comment (as2388) [inline configuration/pipeline_config.json]:
```suggestion
      "ShowPipelineKeyToRemapTo": "diagnostic_s01e03_raw",
```

File: configuration/pipeline_config.json [hunk 5]
```diff
@@ -40,12 +40,26 @@
...
+      "RangeStartInclusive": "2020-08-26T17:00:00+03:00",
+      "RangeEndExclusive": "2020-08-27T13:00:00+03:00"
+    }
+    ],
```
Comment (as2388) [inline configuration/pipeline_config.json]:
```suggestion
  ],
```

File: configuration/pipeline_config.json [hunk 6]
```diff
@@ -40,12 +40,26 @@
...
   "PhoneNumberUuidTable": {
     "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
     "TableName": "IMAQAL"
   },
+  "TimestampRemappings": [
+    {
+      "TimeKey": "received_on",
+      "ShowPipelineKeyToRemapTo":  "diagnostic_s01e03_raw",
+      "RangeStartInclusive": "2020-08-26T17:00:00+03:00",
+      "RangeEndExclusive": "2020-08-27T13:00:00+03:00"
```
Comment (as2388) [inline configuration/pipeline_config.json:60]: The CSV goes to 15:17, are you restricting to 13:00 because we already have those messages anyway?

**Review by as2388 — APPROVED** [review]

File: configuration/pipeline_config.json (hunk 6 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json:60]: Yes the network resumed at 12 but to be safe have updated to 15:17 thanks.

**Pull request merged**